ui/address/contract/ContractRead.tsx

```diff
@@ -38,6 +38,8 @@ const ContractRead = ({ addressHash, isProxy, isCustomAbi }: Props) => {
     },
   });
 
+  const filteredData = data?.filter((item) => item.stateMutability === 'view') || [];
```

Comment:
Add also the other conditions
`item.stateMutability === 'view' || item.stateMutability === 'pure' || item?.constant`

Reply:
@yurikoinaba added the constant one, but `pure` is not a possible state on the [type definition](https://github.com/HorizenLabs/blockscout-frontend/blob/4d9edb4ce43b2a8e1f44f2a1e5c421aecf77cf62/types/api/contract.ts#L4), I think they're always going to be `view`.

I checked the updated source code and it's the same.